libs/Wallet/src/NewWalletAccountController.cpp

```diff
@@ -0,0 +1,190 @@
+#include "Status/Wallet/NewWalletAccountController.h"
+
+#include <StatusGo/Wallet/WalletApi.h>
+
+#include <StatusGo/Accounts/AccountsAPI.h>
+#include <StatusGo/Accounts/Accounts.h>
+#include <StatusGo/Accounts/accounts_types.h>
+#include <StatusGo/Metadata/api_response.h>
+#include <StatusGo/Utils.h>
+#include <StatusGo/Types.h>
+
+#include <Onboarding/Common/Constants.h>
+
+#include <QQmlEngine>
+#include <QJSEngine>
+
+#include <ranges>
+
+namespace GoAccounts = Status::StatusGo::Accounts;
+namespace GoWallet = Status::StatusGo::Wallet;
+namespace UtilsSG = Status::StatusGo::Utils;
+namespace StatusGo = Status::StatusGo;
+
+namespace Status::Wallet {
+
+NewWalletAccountController::NewWalletAccountController(std::shared_ptr<Helpers::QObjectVectorModel<WalletAccount>> accounts)
+    : m_accounts(accounts)
+    , m_mainAccounts(std::move(filterMainAccounts(*accounts)), "account")
+    , m_derivedAddress("derivedAddress")
+    , m_derivationPath(Status::Constants::General::PathWalletRoot)
+{
+}
+
+NewWalletAccountController::~NewWalletAccountController()
+{
+}
+
+using AccountQmlList=QQmlListProperty<WalletAccount>;
```

Comment:
Can't see it anywhere being used? :)

Reply:
I replaced it with `QObjectVectorModel.h` without much time cost-wise but with more performance/maintenance benefits. Thanks for the heads up.